common/shlibs

```diff
@@ -3924,3 +3923,4 @@ libsimavr.so.1 simavr-1.6_2
 libsimavrparts.so.1 simavr-1.6_2
 libsword-1.8.1.so libsword-1.8.1_6
 libtree-sitter.so.0 tree-sitter-0.18.2_1
+libQuotient.so.0.6 libQuotient-0.6.5_1
```

Comment:
I'd put this line below the old one, otherwise you can end up having to rebase the PR constantly due to new packages.